OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -70,20 +70,51 @@ class Device : public ModelComponent {
     /** Add outputs so we can report device quantities we care about. */
     /** The length of the device from anchor to anchor point. */
     OpenSim_DECLARE_OUTPUT(length, double, getLength, SimTK::Stage::Position);
-
-    // TODO: add other outputs.
+    /** The lengthening speed of the device. */
+    OpenSim_DECLARE_OUTPUT(speed, double, getSpeed, SimTK::Stage::Velocity);
+    /** The force transmitted by the device. */
+    OpenSim_DECLARE_OUTPUT(tension, double, getTension, SimTK::Stage::Dynamics);
+    /** The power produced(+)/dissipated(-) by the device. */
+    OpenSim_DECLARE_OUTPUT(power, double, getPower, SimTK::Stage::Dynamics);
+    /** The height of the model that device is attached to. */
+    OpenSim_DECLARE_OUTPUT(height, double, getHeight, SimTK::Stage::Position);
```

Comment:
Unclear how `height` differs from `com_height` (or whether they are meant to be equal). Perhaps `foot_height` or `clearance` is intended?